corehq/apps/data_dictionary/management/commands/populate_case_prop_groups.py

```diff
@@ -13,6 +14,7 @@ def add_arguments(self, parser):
             help="Domain name(s). If blank, will generate for all domains")
 
     def handle(self, **options):
+        remove_out_of_sync_prop_and_groups()
```

Comment:
What's the story here?  An issue in the sync code?

Reply:
This code was added to fix the case where the user changes the group for a property after this merging and migrating this PR but before the Next PR for this change is merged. Here the group field for a property would be out of sync with group_obj reference, so this code resets the group_obj field to None. Then the populate_case_prop_groups function would sync it again.